habitat-lab/habitat/config/default_structured_configs.py

```diff
@@ -1525,6 +1585,7 @@ class AgentConfig(HabitatBaseConfig):
     start_position: List[float] = field(default_factory=lambda: [0, 0, 0])
     start_rotation: List[float] = field(default_factory=lambda: [0, 0, 0, 1])
     joint_start_noise: float = 0.1
+    joint_that_can_control: Optional[List[int]] = None
```

Comment:
Hi Andrew, thanks for this! The reason is that I want to have a robot-agnostic way to decide which joint to control given all joints. In practice, you might have some joints that you do not want to control since that could exceed the robot joint constraint or you want the robot movement to be more stable. For example, in Spot, there are 6 joints, but we only want to control 4 of them since it will be more stable for training mobile gaze skills. For other skills, you want to control all of them since it will be more agile or something. Please let me know what you think, and thanks a lot!